lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +43,23 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str) or text == '':
```

Comment:
Empty sequences are mapped to false anyway, check it simpler

Reply:
Fixed